types/passport-totp/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for passport-totp 0.0
+// Project: https://github.com/jaredhanson/passport-totp
+// Definitions by: Semen Verbych <https://github.com/JoncoLab>
```

Comment:
@saladin-uzh this would require (at)JoncoLab to explicitely give ACK, while it seems to be 
an organization, not a regular user